std/haxe/macro/CompilationServer.hx

```diff
@@ -69,6 +69,16 @@ class CompilationServer {
 		});
 	}
 
+	/**
+		Invalidates a module, removing it from the cache.
+
+		If the module has already been loaded in current context, a compiler
+		error will be raised which can be caught using `try ... catch`.
+	**/
+	static public function invalidateModule(path:String) {
```

Comment:
In general I'm somewhat worried about exposing this to users because it requires some knowledge of how our compilation cache works. Could you tell me in which cases this function is supposed to be used?

Reply:
Most recent one I've seen is a dependency on an external file but only for specific changes (for example a field in a json file, with the file itself changing all the time)

Reply:
That sounds pretty wild... but well, I guess I'll not question it.